platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/AdminConsoleService.java

```diff
@@ -416,6 +418,15 @@ public void updateForLocation(
     }
   }
 
+  private Object sanitizeUIConfiguration(String pid, String configEntryKey, Object configEntryValue) {
+    if (UI_CONFIG_PID.equals(pid)) {
+      if ("color".equals(configEntryKey) || "background".equals(configEntryKey)) {
+        return String.valueOf(configEntryValue).split(";")[0];
```

Comment:
I feel like it's a bit odd to split the input and assume whatever came before a semicolon was valid. If someone is adding semicolons to this field, they're probably doing some sketch stuff. Returning an error to the frontend and not saving any changes at all would be ideal. Or at the very least just stripping out or encoding bad characters.